src/modules/mavlink/mavlink_bridge_header.h

```diff
@@ -55,6 +55,12 @@
 #define MAVLINK_GET_CHANNEL_BUFFER mavlink_get_channel_buffer
 #define MAVLINK_GET_CHANNEL_STATUS mavlink_get_channel_status
 
+#if defined(__PX4_POSIX)
```

Comment:
Even on NuttX some people want more than 4. 

We have a new `CONSTRAINED_MEMORY` define now (analogous to CONSTRAINED_FLASH) that's only enabled on cortex m4.

Reply:
Nice, I'll change it to that.